app/controllers/admin/merchants_controller.rb

```diff
@@ -1,10 +1,34 @@
 class Admin::MerchantsController < ApplicationController
   before_action :require_admin
 
+  def index
+    @merchants = Merchant.all
+  end
+
   def show
     @merchant = Merchant.find(params[:id])
   end
 
+  def disable
+    merchant = Merchant.find(params[:id])
+    merchant.toggle_active
+    merchant.items.update_all(active?: false)
+    merchant.save
+    flash[:notice] = "#{merchant.name} is now disabled"
+
```

Comment:
Maybe we can combine these two to one method?
Not restful naming, but I like how it's specific... Is this ok?